logstash-core/lib/logstash/output_delegator.rb

```diff
@@ -153,10 +168,14 @@ def busy_workers
     if @threadsafe
       0
     else
-      @workers.size - @worker_queue.size
+      @workers.size - (@worker_queue ? @worker_queue.size : 0)
```

Comment:
Ah, I found the reason for this. The pipeline reporter can run before the outputs are registered trying to pull a value here. In that case @worker_queue is empty, we just return 0. I'll add a comment here so it isn't confusing.